charge.go

```diff
@@ -144,6 +167,194 @@ type ChargeLevel3 struct {
 	ShippingAmount     int64                   `json:"shipping_amount"`
 }
 
+// ChargePaymentMethodDetailsAchCreditTransfer represents details about the ACH Credit Transfer
+// PaymentMethod.
+type ChargePaymentMethodDetailsAchCreditTransfer struct {
+	AccountNumber string `json:"account_number"`
+	BankName      string `json:"bank_name"`
+	RoutingNumber string `json:"routing_number"`
+	SwiftCode     string `json:"swift_code"`
+}
+
+// ChargePaymentMethodDetailsAchDebit represents details about the ACH Debit PaymentMethod.
+type ChargePaymentMethodDetailsAchDebit struct {
+	AccountHolderType BankAccountAccountHolderType `json:"account_holder_type"`
+	BankName          string                       `json:"bank_name"`
+	Country           string                       `json:"country"`
+	Fingerprint       string                       `json:"fingerprint"`
+	Last4             string                       `json:"last4"`
+	RoutingNumber     string                       `json:"routing_number"`
+}
+
+// ChargePaymentMethodDetailsAlipay represents details about the Alipay PaymentMethod.
+type ChargePaymentMethodDetailsAlipay struct {
+}
+
+// ChargePaymentMethodDetailsBancontact represents details about the Bancontact PaymentMethod.
+type ChargePaymentMethodDetailsBancontact struct {
+	AccountHolderType BankAccountAccountHolderType `json:"account_holder_type"`
+	BankName          string                       `json:"bank_name"`
+	Country           string                       `json:"country"`
+	Fingerprint       string                       `json:"fingerprint"`
+	Last4             string                       `json:"last4"`
+	RoutingNumber     string                       `json:"routing_number"`
+}
+
+// ChargePaymentMethodDetailsBitcoin represents details about the Bitcoin PaymentMethod.
+type ChargePaymentMethodDetailsBitcoin struct {
+	Address        string `json:"address"`
+	Amount         int64  `json:"amount"`
+	AmountCharged  int64  `json:"amount_charged"`
+	AmountReceived int64  `json:"amount_received"`
+	AmountReturned int64  `json:"amount_returned"`
+	RefundAddress  string `json:"refund_address"`
+}
+
+// ChargePaymentMethodDetailsCardThreeDSecure represents details about 3DS associated with the
+// charge's PaymentMethod.
+type ChargePaymentMethodDetailsCardThreeDSecure struct {
+}
+
+// PaymentMethodDetailsCardWallet represents the details of the card wallet if this Card
+// PaymentMethod is part of a card wallet.
+type PaymentMethodDetailsCardWallet struct {
```

Comment:
@remi-stripe  should this be prefixed with `ChargeXXX` like other structs too?

Reply:
Yes it should, nice catch.